README.md

```diff
@@ -692,31 +692,29 @@ are exposed on the `ghw.PCIInfo` struct.
 The `ghw.PCI()` function returns a `ghw.PCIInfo` struct. The `ghw.PCIInfo`
 struct contains a number of fields that may be queried for PCI information:
 
+* `ghw.PCIInfo.Devices` is a slice of pointers to `ghw.PCIDevice` structs that
```

Comment:
Is not clear to me if calling `ListDevices()` is now deprecated or still fully supported. Both approaches works from client perspective. As API consumer I for myself prefer to have functions as part of the APIs (e.g. calling ListDevices() explicitely to get a fresh list of devices) but this is mostly personal taste.

Reply:
I replaced ListDevices() with a PCIInfo.Devices slice to make the interface match the other "XXXInfo" structs for consistency purposes. I've gone ahead and deprecated the ListDevices exported function and will unexport it in the v1.0 release (as of yet, I don't have a release date target for v1.0 though) :)